### ISSUE TITLE
Update GLideN64 again

### ISSUE THREAD

**Issue by thecoreyburton** [TASVideos/BizHawk#1053]
There have been a number of changes and improvements since the last implementation, some of which are very relevant to [this submission](http://tasvideos.org/5678S.html). Could the plugin please be updated once more?

Additionally, there are a number of bugs that are present in the Bizhawk version GLideN64 that aren't in the same version from the official repo and there is a chance this could be one of them. Unfortunately, I won't know for certain until it's implemented.

**Comment by zeromus** [comment]
dup #711

**Comment by zeromus** [comment]
or here, #879. Future threads will be terminated with prejudice.

**Comment by thecoreyburton** [comment]
Ah, my apologies for that. It's easy to forget Bizhawk is a multi-platform emulator and so, whilst in my head they're three separate situations, it would have probably been a better idea to list them all under a general N64 update issue.

I only requested this update as recent changes made to the plugin itself help correctly emulate a run I'm in the process of publishing. I'd be happy to handle such plugin updates myself, only when I tried to do so I got [this](https://imgur.com/a/qQEdt) as the output and I'm not sure what modifications would be needed in order to fix it.

Regarding the duplicates, I was hoping that swapping out the GLideN64 plugin for a more recent version would be a smaller update in contrast to overhauling the whole core as discussed in #879, and #711 pertains to the audio plugin and (as far as I know), the two shouldn't have had any direct impact on one-another. In either case, now that #879 has been altered and has a potential fix for #711, you're welcome to close the latter too and leave #879 as a general thread for issues regarding the N64 core in Bizhawk (and it's associated plugins).

Again, sorry about any clutter or inconvenience. It's very easy as a user to forget the scope of the project and the work your requests might take.
